src/main/java/Monday.java

```diff
@@ -0,0 +1,98 @@
+import java.util.Scanner;
+
+public class Monday {
+    private static final String LOGO = " __  __                 _             \n"
+            + "|  \\/  | ___  _ __   __| | __ _ _   _ \n"
+            + "| |\\/| |/ _ \\| '_ \\ / _` |/ _` | | | |\n"
+            + "| |  | | (_) | | | | (_| | (_| | |_| |\n"
+            + "|_|  |_|\\___/|_| |_|\\__,_|\\__,_|\\__, |\n"
+            + "                                |___/  \n";
+    private static final String LINE = "____________________________________________________________";
+    private Task[] tasks = new Task[100];
+    private int taskCount = 0;
+
+    public static void main(String[] args) {
+        Monday monday = new Monday();
+        monday.run();
+    }
+
+    private void run() {
+        printWelcomeMessage();
+        handleUserInput();
+    }
+
+    private void printWelcomeMessage() {
+        System.out.println("Hello from\n" + LOGO);
+        System.out.println(LINE);
+        System.out.println("    Hello! I'm MONDAY\n    What can I do for you?");
+        System.out.println(LINE);
+    }
+
+    private void handleUserInput() {
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        while (true) {
+            input = scanner.nextLine();
+            System.out.println(LINE);
+
+            if (input.equalsIgnoreCase("list")) {
+                listTasks();
+            } else if (input.startsWith("mark ")) {
+                markTaskAsDone(input);
+            } else if (input.startsWith("unmark ")) {
+                unmarkTaskAsNotDone(input);
+            } else if (input.equalsIgnoreCase("bye")) {
+                printGoodbyeMessage();
+                break;
+            } else {
+                addTask(input);
+            }
```

Comment:
I like the clear separation of concerns using different methods, all with descriptive names !